dev/ci/ci-waterproof.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env/ bash
+
+set -e
+
+ci_dir="$(dirname "$0")"
+. "{ci_dir}/ci-common.sh"
+
+git_download waterproof
+
+if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
+
+( cd "${CI_BUILD_DIR}/coq-waterproof" 
```

Comment:
```suggestion
( cd "${CI_BUILD_DIR}/waterproof" 
```
the directory name comes from the name we give to `git_download` not the name in the url